docs: toggle button guide adapted (#15)
siemens/ix-docs: docs/components/toggle-button/guide.md

## Behavior in context

- **Independent toggling:** Toggle buttons are typically used on their own or in layouts where each button represents an independent setting or mode. For example, toggling bold, italic, or underline in a text editor.

## States

Toggle buttons have five states: Default, hover, active, disabled, loading and focused. All states are also available for pressed toggled buttons.

## Dos and Don'ts

- Do use toggle buttons when users can switch a setting on or off independently
- Do use toggle buttons when two opposing options don't follow the on/off metaphor

> - Don’t use toggle buttons in button groups where only one option can be selected, here standard buttons are used instead

Reviewer: ```suggestion
- Don’t use toggle buttons in button groups where only one option can be selected (use normal [buttons](../button/index.mdx) or [icon buttons](../icon-button/index.mdx) instead)
```
